AutomationDemo/features/login.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Twilio Conversations Demo App
+
+  Scenario Outline: As a user, I can log into the Twilio Conversations Demo App using my credentials
+
+    Given I am on the login page
+    When I login with <username> and <password>
+    Then I should see available conversations
+
+    Examples:
+      | username         | password             |
+      | <your usernam>   | <your password>      |
```

Comment:
can we specfiy those through env/context variables somehow?

Reply:
I think this should be possible to do, but could not yet find a sure-fire way how to do so, so I think a ticket should be made for me to investigate this after my PTO.